### ISSUE TITLE
Any plans for a Sample project?

### ISSUE THREAD

**Issue by lapastillaroja** [pardom/Ollie#37]
It would be a life saver :)

**Comment by Saketme** [comment]
Also, more comments throughout the public methods please.

**Comment by pardom** [comment]
Yes, there are plans. I don't have a lot of time to work on Ollie at the moment, but I will open the wiki up soon with some stubbed out docs to fill in.

**Comment by Rainer-Lang** [comment]
@pardom Thanks. This could help. :-)

**Comment by niranjan94** [comment]
@lapastillaroja  for now, you can see https://github.com/pardom/Ollie/tree/master/core/src/test/java/ollie/test for code samples
